core/src/test/java/com/google/cloud/sql/core/SqlAdminApiFetcherTest.java

```diff
@@ -39,6 +39,8 @@ public class SqlAdminApiFetcherTest {
 
   public static final String SAMPLE_PUBLIC_IP = "34.1.2.3";
   public static final String SAMPLE_PRIVATE_IP = "10.0.0.1";
+  public static final String SAMPLE_PSC_IP = "10.0.0.2";
+  public static final String SAMPLE_PSC_HOSTNAME = "abcde.12345.us-central1.sql.goog";
```

Comment:
thoughts on maybe re-naming `SAMPLE_PSC_HOSTNAME` to `SAMPLE_PSC_DNSNAME` to be a bit more accurate?

Reply:
Changed to SAMPLE_PSC_DNS_NAME.